network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java

```diff
@@ -54,6 +54,9 @@ public class SvgParameters {
     private int angleValuePrecision = 1;
     private double pstArrowHeadSize = 8;
 
+    /** em dash unicode for undefined value */
+    private String undefinedValueSymbol = "\u2014";
```

Comment:
Could you replace this by `""`? it doesn't really look good a dash above an edge, and now we can take advantage of this new feature!

Reply:
Yes, my bad, I forgot we talked about that !